templates/checkout/index.html

```diff
@@ -80,6 +80,9 @@
           <a href="{% url "checkout:login" %}" class="btn primary float-right cart__submit">
             {% trans "Checkout" context "Cart primary action" %}
           </a>
+          <button class="btn secondary float-right cart__clear" action="{% url 'cart:clear-cart' %}">
+            {% trans "Clear Cart" context "Clear Cart" %}
```

Comment:
Could you add a more meaningful context? Could be "Cart summary page: secondary action, empty the cart".